drafts/proof-of-spacetime.md

```diff
@@ -1,24 +1,141 @@
 ## Proof-of-Spacetime
-   
-This document descibes 
- 
+
+This document descibes
+
+- Base-PoSt
 - VDF-PoSt: a Proof-of-Spacetime using VDFs
 - An Extension to PoSt to support multiple sectors
-- An Extension to PoSt to support challenges taken from a Random Beacon 
+- An Extension to PoSt to support challenges taken from a Random Beacon
 
 ## Syntax
 
 - **PoSt Epoch**: The total time passing between Online PoReps in the PoSt Computation (in VDF, this interval is the time it takes to run a VDF and an Online PoRep prove step). We define the number of epochs as `POST_EPOCHS_COUNT`
-- **PoSt Period**: The total time it takes to run a single PoSt. If a PoSt is repeated multiple times, we define the number of periods as `POST_PERIODS_COUNT`. This can be reasoned as: `PoSt Epoch * POST_EPOCHS_COUNT`. We assume that the best  Post Period time is `MIN_POST_PERIOD_TIME` 
+- **PoSt Period**: The total time it takes to run a single PoSt. If a PoSt is repeated multiple times, we define the number of periods as `POST_PERIODS_COUNT`. This can be reasoned as: `PoSt Epoch * POST_EPOCHS_COUNT`. We assume that the best  Post Period time is `MIN_POST_PERIOD_TIME`
 - **Total proving time**: The time it takes to run a PoSt. Note that a PoSt could be a composition of multiple PoSt. This can be reasoned as: `PoSt Period * POST_PERIODS_COUNT` TODO Check with papers' syntax
```

Comment:
The current incarnation of the [specification](https://github.com/filecoin-project/specs/blob/master/actors.md#submitpost) shows that "proving period duration" is a function of sector size. I assume that `Total proving time` and proving period duration are the same thing. So, I have some questions (pertaining to our PoSt construction which uses the random beacon):

- Will `POST_PERIODS_COUNT` be fixed for all sector sizes?
- Will `PoSt Period` be a function of sector size?
- Will `PoSt Epoch` be a function of sector size?

Reply:
- epoch and periods_count are no more
- very likely period will _not_ be a function of the sector size, but this is still being discussed

Reply:
most of those words are gone, relationship to sector sizes will come in a future pr